frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/moby/buildkit/util/apicaps"
 	binfotypes "github.com/moby/buildkit/util/buildinfo/types"
 	"github.com/moby/buildkit/util/gitutil"
+	"github.com/moby/buildkit/util/imageutil"
```

Comment:
`imageutil` imports many containerd libraries and isn't ideal to be imported here. Move the image types to a separate package that is imported both by dockerfile and imageutil.

Reply:
@tonistiigi thanks for your review, I have move the Image definition to a new package under `util/schema1`, PTALA.